src/views/v2/month/day-events-multiday.php

```diff
@@ -0,0 +1,39 @@
+<?php
+/**
+ * View: Month View - Day Events Multiday
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/events/views/v2/month/day-events-multiday.php
+ *
+ * See more documentation about our views templating system.
+ *
+ * @link {INSERT_ARTCILE_LINK_HERE}
+ *
+ * @version TBD
+ *
+ */
+$day_number = $this->get( 'day' );
+$month      = $this->get( 'month' );
+
+// Get the multiday events for that day
+// @todo: This is a function with demo purposes.
+$events_multiday = tribe_events_views_v2_month_demo_day_get_events_multiday( $month, $day_number );
+
+if ( ! $events_multiday ) {
+	return;
+}
+?>
+
+<?php foreach ( $events_multiday as $event ) : ?>
```

Comment:
If you're putting a wrapper around `day-events`, then we should put a wrapper around `day-events-multiday` as well to keep consistent. But we also need to consider the different spacing cases here and think about whether having a wrapper for both makes sense or not.